.husky/exec-git-cz.cmd

```diff
@@ -0,0 +1,19 @@
+@ECHO off
```

Comment:
Looks only *.cmd files checked in, will it work on Linux/macOS?

Reply:
default will run by prepare-commit-msg file, in prepare-commit-msg will judge which platform, windows will run cmd, and *nix will run prepare-commit-msg file

Reply:
got it, thanks for the explanation.